external/coreclr/coreclr.depproj

```diff
@@ -3,6 +3,8 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <OutputPath>$(RuntimePath)</OutputPath>
+     <!-- WORKAROUND: Force external packages to be restored for x64 until arm packages are fully broughtup-->
+    <NugetRuntimeIdentifier Condition="'$(ArchGroup)' == 'arm'">$(RuntimeOS)-x64</NugetRuntimeIdentifier>
```

Comment:
I'm good with this can you please add an issue and link that in the comment. so we remember to fix this once it is brought-up.